### ISSUE TITLE
Error when fetch used with data-url

### ISSUE THREAD

**Issue by mixtur** [DevExpress/testcafe#2865]
### Are you requesting a feature or reporting a bug?

bug

### What is the current behavior?

testcafe throws an error
`Uncaught TypeError: Cannot read property 'destUrl' of null`

### What is the expected behavior?

no error

### How would you reproduce the current behavior (if this is a bug)?

use fetch with data-url in tested code.
For example:

```html
<script>
    fetch('data:text/plain,foo')
        .then(x => x.text())
        .then(console.log);
</script>
```

Any test will fail including empty
For example:

```js
fixture`bug`
    .page`./index.html`;

test('empty test', async () => {});
```

### Specify your

* operating system: Ubuntu 16.04
* testcafe version: 0.22.0
* node.js version: 10.9.0

**Comment by miherlosev** [comment]
Hi @mixtur 

At present, TestCafe does not support calling `fetch` with data urls. I've marked this issue as enhancement.

**Comment by miherlosev** [comment]
The fix was published in `testcafe@0.22.1-alpha.3`

**Comment by mixtur** [comment]
thx
